src/PhoneNumber/PhoneNumber.js

```diff
@@ -84,73 +57,27 @@ class PhoneNumber extends Component {
   };
 
   handleChange = e => {
-    const { value: currentNumber, selectionStart: caretPosition } = e.target;
-    const { phonePattern, value: selectedCountry } = this.getCountryFrom(
-      currentNumber
-    );
+    const { value: currentNumber } = e.target;
+    const { prefix } = this.state;
 
-    let formattedNumber = currentNumber.replace(/(?!^\+)\D/gm, '');
+    const phoneNumber = currentNumber.replace(/[^\d]/gm, '');
 
-    if (phonePattern) {
-      formattedNumber = this.formatNumber(formattedNumber, phonePattern);
-    }
+    const formattedNumber = prefix ? `+${prefix} ${phoneNumber}` : phoneNumber;
```

Comment:
This is line is much better now! 
However, i still miss testing this feature

Reply:
Ok